versioned_docs/version-v2/framework-integration/react.md

```diff
@@ -75,18 +75,12 @@ If you do rename your React component library, be sure to change the `name` in t
 
 ### Install the React Output Target in your Stencil Component Library
 
-Now that the project structure is set up, we can install the React Output Target package in your Stencil component library. This package contains the React wrapper function that we will use to generate our React wrapped components inside a 'React component library'. To install the React Output Target package, run the following command in your **Stencil project directory**
+Now that the project structure is set up, we can install the React Output Target package in your Stencil component library. This package contains the React wrapper function that we will use to generate our React wrapped components inside a 'React component library'. To install the React Output Target package, run the following command in your **Stencil project directory**:
```

Comment:
Looks like we're adding a colon here for the v2 docs, but nowhere else. Is this intentional?

Reply:
I think I just did that on a whim and forgot to do so elsewhere, I'll pop that in the other files too